Utilities/Config.h

```diff
@@ -309,6 +312,73 @@ namespace cfg
 		}
 	};
 
+	// Float entry with custom Min/Max range.
+	template <s64 Min, s64 Max>
+	class _float final : public _base
+	{
+		static_assert(f64{Min} < f64{Max}, "Invalid cfg::_float range");
+
+		// Prefer 32 bit type if possible
```

Comment:
I doubt there is a reason to prefer float, due to accuracy loss in mantissa.

Reply:
Okay.